manifests/base/alertmanager/alert_rules.yaml

```diff
@@ -31,4 +31,9 @@ data:
             clusterID: "{{ $labels.clusterID }}"
             PersistentVolumeClaim: "{{ $labels.persistentvolumeclaim }}"
             severity: warning            
-
+        - alert: CriticalPolicyReport
+          annotations:
+            summary: There is a policy report with severity of critical.
+            description: "The policy: {{ $labels.policy }} has a severity of critical"
+          expr: sum(acm_policyreport_info{severity="critical"}) by (cluster,category, namespace, pod, policy)
+          for: 1m
```

Comment:
@RobDolares :
We should add the `severity: critical `to this alert rule
Also please check if the `namespace, pod information` really captures the offending pod/namespace or is just prom monitoring details. Cannot tell without looking at the real data.
Is it reasonable to add clusterid to the `by (cluster...) `as well as add `cluster, clusterid `to the description so that customers have an easy way of know which clusters are affected?

Reply:
Disaiah and I were looking at those options and the redundancy when a visual alert is configured.  That is why the existing filters are applied.  But I agree with clusterID and will update.

As for namespace and pod - you are correct, it is just prom monitoring deets and we can remove those.